build.rs

```diff
@@ -41,14 +41,27 @@ fn main() {
 
     std::env::set_var("FISH_BUILD_VERSION", version);
 
-    #[cfg(feature = "installable")]
+    let cman = std::fs::canonicalize(env!("CARGO_MANIFEST_DIR")).unwrap();
+    let targetman = cman.as_path().join("target").join("man");
+
+    #[cfg(feature = "embed-data")]
     #[cfg(not(clippy))]
     {
-        let cman = std::fs::canonicalize(env!("CARGO_MANIFEST_DIR")).unwrap();
-        let targetman = cman.as_path().join("target").join("man");
         build_man(&targetman);
     }
+    #[cfg(any(not(feature = "embed-data"), clippy))]
+    {
+        let sec1dir = targetman.join("man1");
+        let _ = std::fs::create_dir_all(sec1dir.to_str().unwrap());
+    }
+
     rsconf::rebuild_if_paths_changed(&["src", "printf", "Cargo.toml", "Cargo.lock", "build.rs"]);
+
+    // These are necessary if built with embedded functions,
+    // but technically only in release builds (because debug builds read from the filesystem).
+    #[cfg(feature = "embed-data")]
+    rsconf::rebuild_if_paths_changed(&["doc_src", "share"]);
```

Comment:
Is rust-embed not usable in debug builds?
Maybe avoid rebuilds for debug builds then:

	#[cfg(all(feature = "embed-data"), debug_assertions)]